src/io/ExportContext.hpp

```diff
@@ -15,7 +15,7 @@ struct ExportContext {
   std::string location;
 
   // @brief Exporting timestep interval (equals -1 when not set).
-  int timestepInterval;
+  int timeWindowInterval;
```

Comment:
Documentation and variable name do not really match the purpose. I would suggest

```
  // @brief Frequency of export. It is only possible to export at the end of a time window (equals -1 when not set).
  int outputFrequency
```